osvr/RenderKit/DirectModeVendors.h

```diff
@@ -241,6 +241,7 @@ namespace renderkit {
             Vendor{"TVL", "TotalVision"},
             Vendor{"FOV", "FOVE"},
             Vendor{"LZT", "HomeVR"},
+            Vendor{"NEC", "Lumus"},
```

Comment:
Is this vendor ID specific to your display, or is this going to put all NEC displays into DirectMode?

Reply:
It is specific to the particular model of the display that I'm using. Is there a problem if all NEC displays are put into DirectMode because of this? If that is so, is there a way to avoid this?

Reply:
The problem is that all displays, even LCD screens and projectors, will get put into DirectMode.  We haven't had a case yet where a major screen vendor would have all of its displays changed, so far it has been small screen developers that make components for HMDs.  There is not a way to avoid it in the current code base.  @rpavlik what do you think about the pull request, given the risk?

Reply:
Okay, so my end goal here is to get the display working in DirectMode on Intel Graphics. Given that enabling 'NEC' in the RenderManager source can have unwanted implications, I've been looking for alternative solutions. After a bit of searching around, I think I might have a temporary solution to test things out first. Using a pass-through EDID modifier like https://www.hdfury.com/product/dr-hdmi/ I should be able to change the EDID advertised by the display to one of the white-listed vendor IDs. If I then plug into an Intel Graphics HDMI port, DirectMode should work for me, right?

Reply:
That seems riskier than putting the correct one in.  I think it will work, but will probably confuse autodiscovery either now or in the future.

Reply:
This would have been something to add to the "non-default" list below. Probably should be revised now.

Adding the entry is not needed - you can pass the PNPID in as the Vendor Name in the display descriptor, and to the enable/disable utils - though the latter do not exist for intel direct mode (not sure if that's for functional/api reasons or NDA reasons).

Reply:
@rpavlik Added in pull request https://github.com/sensics/OSVR-RenderManager/pull/361